core/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/MessageConsumer.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.core.util;
+
+import java.io.Closeable;
+import java.io.IOException;
+
+/**
+ * Consumes messages one by one for analysing, writing, storing, etc.
+ *
+ * @author Alexander Garagatyi
+ */
+public interface MessageConsumer<T> extends Closeable {
+    void write(T message) throws IOException;
```

Comment:
It is obvious what this single  method  in this class does. So I would prefer to omit useless javadocs here.